src/Cargo.toml

```diff
@@ -58,3 +59,11 @@ debug-assertions = false
 
 [patch.'https://github.com/rust-lang/cargo']
 cargo = { path = "tools/cargo" }
+
+# Override rustfmt dependencies both on the repo and the crate (the RLS
+# sometimes uses either).
+# FIXME should only need the crates.io patch, long term.
+[patch.'https://github.com/rust-lang-nursery/rustfmt']
```

Comment:
single quotes confused some editors (e.g. the vscode toml plugin) - could you make them double quotes?